.dev/tests/cypress/helpers.js

```diff
@@ -233,14 +233,30 @@ export function setColorSetting( settingName, hexColor ) {
 		case 'text':
 			cy.get( '.components-base-control__field' )
 				.contains( /text color/i )
-				.then( $backgroundPanel => {
-					cy.get( Cypress.$( $backgroundPanel ).parent() )
+				.then( $textPanel => {
+					cy.get( Cypress.$( $textPanel ).parent() )
 						.contains( /custom color/i )
 						.click();
 					cy.get( '.components-color-picker__inputs-field input[type="text"]' )
 						.clear()
 						.type( hexColor );
-					cy.get( Cypress.$( $backgroundPanel ).parent() )
+					cy.get( Cypress.$( $textPanel ).parent() )
+						.contains( /custom color/i )
+						.click();
+				} );
+			break;
+
+		case 'color':
+			cy.get( '.components-base-control__field' )
+				.contains( /color/i )
+				.then( $colorPanel => {
+					cy.get( Cypress.$( $colorPanel ).parent() )
+						.contains( /custom color/i )
+						.click();
+					cy.get( '.components-color-picker__inputs-field input[type="text"]' )
+						.clear()
+						.type( hexColor );
+					cy.get( Cypress.$( $colorPanel ).parent() )
```

Comment:
@EvanHerman I rewrote this helper function in my phase 3 tests. Please switch to this alternate helper function. which reduces the footprint and is more simplistic. 

https://github.com/godaddy-wordpress/coblocks/blob/644f045cf8743a87de80b85a478935613f45f73b/.dev/tests/cypress/helpers.js#L244-L265